dso/model.py

```diff
@@ -508,6 +514,21 @@ class DikiFinding(Finding):
     def key(self) -> str:
         return _as_key(self.provider_id, self.ruleset_id, self.rule_id)
 
+@dataclasses.dataclass(frozen=True)
+class GitHubSecretFinding(Finding):
+    secret_type: str
+    path: str
+    line: int
+    state: str
+    created_at: str
+    updated_at: str
+    repository: str
+
+    # every finding has a key with repo+path+line+secret type
+    @property
+    def key(self) -> str:
+        return _as_key(self.repository, self.path, str(self.line), self.secret_type)  
```

Comment:
I think it should be fine to not include github-host, considering we also have ocm-component + source (although we _might_ consider including it, just to be sure).

However, I think it might be worthwhile considering to include finding-ID (I think I remember such an ID from the example you showed last time) into `key`. Rationale: it might happen that a different secret will be found in the same path, at the same line. In which (likely very rare) case I think it will be better to be able to tell those apart.
WDYT?

Reply:
Yes, I agree with you. 
Adding the html_url and secret to the findings. Actually this is the first draft, the new findings format is not clear, let's see. Thanks for taking time to look at this. 😃 
```
"html_url": "https://github.com/kubernetes/demo/security/secret-scanning/123"
```